src/sysext/sysext.c

```diff
@@ -91,6 +98,66 @@ static const struct {
         }
 };
 
+static bool need_reload(void) {
+        // Parse the mounted images to find out if we need
+        // to reload the daemon.
+        _cleanup_free_ char *f = NULL, *buf = NULL, **mounted_extensions = NULL;
+        int r;
+
+        STRV_FOREACH(p, arg_hierarchies) {
+            f = path_join(*p, image_class_info[arg_image_class].dot_directory_name, image_class_info[arg_image_class].short_identifier_plural);
+            if (!f)
+                    continue;
+
+            r = read_full_file(f, &buf, NULL);
+            if (r < 0)
+                    continue;
```

Comment:
it might be ok sometimes to eat up errors, but if so, plesae always debug log.

that said, I don#t think this is a case where we should eat up errors. it's our own setup here, and if it is broken this should cause a loud failure, not just a silent going on